### PR TITLE
feat(add coverage percentage check)

### PR DESCRIPTION
**Pull request by neoshadybeat**

### CODE REVIEW

**Comment by neoshadybeat** [comment]
Need to add more unit tests, waiting for code review

File: .gitignore [hunk 1]
```diff
@@ -2,6 +2,7 @@ composer.phar
 vendor/
 bin/
 php-git-hooks.yml
+.idea
```
Comment (koprivajakub) [inline .gitignore]: put this in your global gitignore.
environment specific folders and files should not be in project gitignore

File: src/PhpGitHooks/Application/PhpUnit/PhpUnitHandler.php [hunk 2]
```diff
@@ -46,7 +46,7 @@ public function run()
         }
     }
 
-    private function setTitle()
+    protected function setTitle()
```
Comment (koprivajakub) [inline src/PhpGitHooks/Application/PhpUnit/PhpUnitHandler.php]: why is this protected now? you override the function anyway in the `PhpUnitCheckCodeCoverageHandler` class

File: src/PhpGitHooks/Tests/Application/Composer/PreCommitProcessorTest.php [hunk 3]
```diff
@@ -28,8 +28,8 @@ protected function setUp()
     public function preCommitHookEnabled()
     {
         $this->IO->shouldReceive('ask')
-            ->times(14)
-            ->andReturn('y', 'y', 'y', 'y', 'y', 'y', 'PSR2', 'y', 'y', 'psr0');
+            ->times(16)
+            ->andReturn('y', 'y', 'y', 'y', 'y', 'y', 'PSR2', 'y', 'y', 'psr0','y');
```
Comment (koprivajakub) [inline src/PhpGitHooks/Tests/Application/Composer/PreCommitProcessorTest.php]: missing whitespace before last `'y'`

File: src/PhpGitHooks/Application/CodeCoverage/CheckCodeCoverageConfigData.php [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+        } else {
+            $answer = $this->configData[self::TOOL]['enabled'];
+        }
+
+        $this->enableTool($answer);
+    }
+
+    private function setMinimunCoveragePercentage()
+    {
+        $percentage = strtolower($this->setQuestion(
```
Comment (koprivajakub) [inline src/PhpGitHooks/Application/CodeCoverage/CheckCodeCoverageConfigData.php]: There is no check if user insert valid value (between 0 and 100)

**Pull request closed without merging**